qa/suites/rgw/singleton/all/rgw_cephadm.yaml

```diff
@@ -0,0 +1,6 @@
+roles: [mon.a, osd.0, osd.1, osd.2, rgw.z.r.a, rgw.z.r.b]
+tasks:
+- cephadm:
+- exec:
+    mon.a:
+    - sleep 30
```

Comment:
it doesn't look like anybody tested this before merging. it's been failing consistently in teuthology/task/kernel.py, but i don't understand the connection to kernel.py
```
2021-04-12T20:22:31.262 DEBUG:teuthology.orchestra.run.smithi033:> rpm -q kernel --last | head -n 1
2021-04-12T20:22:31.394 INFO:teuthology.orchestra.run.smithi033.stdout:kernel-4.18.0-240.22.1.el8_3.x86_64           Mon 12 Apr 2021 08:22:08 PM UTC
2021-04-12T20:22:31.396 DEBUG:teuthology.task.kernel:get_latest_image_version_rpm: 4.18.0-240.22.1.el8_3.x86_64
2021-04-12T20:22:31.396 INFO:teuthology.task.kernel:Not newest distro kernel. Current: 4.18.0-240.10.1.el8_3.x86_64 Expected: 4.18.0-240.22.1.el8_3.x86_64
2021-04-12T20:22:31.397 ERROR:teuthology.run_tasks:Saw exception from tasks.
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_2713a3cd31b17738a50039eaa9d859b5dc39fb8a/teuthology/run_tasks.py", line 91, in run_tasks
    manager = run_one_task(taskname, ctx=ctx, config=config)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_2713a3cd31b17738a50039eaa9d859b5dc39fb8a/teuthology/run_tasks.py", line 70, in run_one_task
    return task(**kwargs)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_2713a3cd31b17738a50039eaa9d859b5dc39fb8a/teuthology/task/kernel.py", line 1247, in task
    (role_remote,) = ctx.cluster.only(role).remotes.keys()
ValueError: too many values to unpack (expected 1)
2021-04-12T20:22:31.658 ERROR:teuthology.run_tasks: Sentry event: https://sentry.ceph.com/organizations/ceph/?query=a96ff9124af145a5a334f39083445e0c
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_2713a3cd31b17738a50039eaa9d859b5dc39fb8a/teuthology/run_tasks.py", line 91, in run_tasks
    manager = run_one_task(taskname, ctx=ctx, config=config)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_2713a3cd31b17738a50039eaa9d859b5dc39fb8a/teuthology/run_tasks.py", line 70, in run_one_task
    return task(**kwargs)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_2713a3cd31b17738a50039eaa9d859b5dc39fb8a/teuthology/task/kernel.py", line 1247, in task
    (role_remote,) = ctx.cluster.only(role).remotes.keys()
ValueError: too many values to unpack (expected 1)
```

i raised https://tracker.ceph.com/issues/50338 against the teuthology project